frontend/src/mixins/FormatDurationUtc.ts

```diff
@@ -3,9 +3,13 @@ import moment from "moment";
 
 @Component
 export default class FormatDurationUtc extends Vue {
-  formatDurationUtc(duration: any, format: string) {
+  formatDurationUtc(
```

Comment:
This function is not used in the current branch code. Do you think we should generalize the function as a part of this task? I doubt. Let's put off these changes until they are really needed by other code. So just revert changes of this file.

Reply:
Except if you agree to merge mixins (see below), then changes would be justified.